proto/interchain_security/ccv/provider/v1/provider.proto

```diff
@@ -106,6 +106,10 @@ message ConsumerAdditionProposal {
   repeated string allowlist = 18;
   // Corresponds to a list of provider consensus addresses of validators that CANNOT validate the consumer chain.
   repeated string denylist = 19;
+  // Corresponds to the minimal amount of (provider chain) stake required to validate on the consumer chain.
```

Comment:
This applies irrespectively of whether the `allow_inactive_vals` flag is set or not, right?

Reply:
Yes!